guides/source/active_record_callbacks.md

```diff
@@ -289,86 +609,93 @@ The following methods trigger callbacks:
 * `update`
 * `update!`
 * `valid?`
+* `validate`
 
-Additionally, the `after_find` callback is triggered by the following finder methods:
+Additionally, the `after_find` callback is triggered by the following finder
+methods:
 
 * `all`
 * `first`
 * `find`
 * `find_by`
+* `find_by!`
 * `find_by_*`
 * `find_by_*!`
 * `find_by_sql`
 * `last`
+* `sole`
+* `take`
 
-The `after_initialize` callback is triggered every time a new object of the class is initialized.
+The `after_initialize` callback is triggered every time a new object of the
+class is initialized.
 
-NOTE: The `find_by_*` and `find_by_*!` methods are dynamic finders generated automatically for every attribute. Learn more about them at the [Dynamic finders section](active_record_querying.html#dynamic-finders)
+NOTE: The `find_by_*` and `find_by_*!` methods are dynamic finders generated
+automatically for every attribute. Learn more about them in the [Dynamic finders
+section](active_record_querying.html#dynamic-finders).
 
 Skipping Callbacks
```

Comment:
Why are the references to these methods removed here? I think they should be there, they are quite useful.

It would be great if there was an additional example here as in the other sections, calling any of these methods and showing that the callbacks are not executed.

Also, what do you think about adding the supress method? It is quite useful. Just make it clear that it only suppresses the saves. [ActiveRecord::Suppressor](https://api.rubyonrails.org/classes/ActiveRecord/Suppressor.html)

Reply:
> Why are the references to these methods removed here? 

@brunoprietog what references are you referring to?

> It would be great if there was an additional example here as in the other sections, calling any of these methods and showing that the callbacks are not executed.

We cover the examples in the section above "Available Callbacks", or did you have any specific scenarios that you had in mind that are not covered by the above section?

> Also, what do you think about adding the supress method? It is quite useful. Just make it clear that it only suppresses the saves. [ActiveRecord::Suppressor](https://api.rubyonrails.org/classes/ActiveRecord/Suppressor.html)

I'm not quite sure about this one, @carlosantoniodasilva do you have thoughts on whether it should be added?

Reply:
> @brunoprietog what references are you referring to?

I'm referring to the links of each method that is in the list of methods that ignores callbacks. Now there are no links to the API reference and before there were.

> We cover the examples in the section above "Available Callbacks", or did you have any specific scenarios that you had in mind that are not covered by the above section?

I mean an example calling a method that ignores callbacks and demonstrating that it does not execute them. In the section on skipping callbacks there are no examples and perhaps the existence of one would be useful in my opinion.

Regarding the suppressor, it was a total unknown to me until I read about it in [this article](https://dev.37signals.com/globals-callbacks-and-other-sacrileges/) on the 37signals dev blog. That's why I think it could be useful.

Reply:
> Regarding the suppressor, it was a total unknown to me until I read about it in [this article](https://dev.37signals.com/globals-callbacks-and-other-sacrileges/) on the 37signals dev blog. That's why I think it could be useful.

I read through the article and the docs and decided to add a section called [Suppressing Callbacks](https://github.com/rails/rails/pull/51654/commits/d9490a0a85bb6e6f06af91cd4d4d1e0fdb89dbe6) which hopefully encapsulates its value while emphasizing cautiousness.